troutstation/code/modules/research/experimentor.dm

```diff
@@ -0,0 +1,649 @@
+/datum/relic_trans
+	var/datum/relic_node/next_node
+	var/desc = "An unknown stimulus."
+
+/datum/relic_trans/proc/check_cond(...)
+		return TRUE
+/datum/relic_trans/proc/get_user(mob/user)
+		return user
+
+/datum/relic_trans/none //Automatically passes - lowest priority (checks other conditions, only procs if nothing has gone on)
+	desc = "None, it is unstable!"
+/datum/relic_trans/touch
+	desc = "A(n) (un)willing participant's touch."
+/datum/relic_trans/harm
+	desc = "Robusting it."
```

Comment:
while robusting is a term that makes a bunch of sense as a ss13 term I did personally prefer thwacking, but that's not really a change I'm asking to make here, just an observation. It's probably fine to stay